app/models/slot_range.rb

```diff
@@ -2,4 +2,8 @@ class SlotRange < ApplicationRecord
   belongs_to :user
   has_many :slots, inverse_of: :slot_range, dependent: :destroy
   accepts_nested_attributes_for :slots, allow_destroy: true
+
+  def display_title
+    "#{from.strftime('%d %B %Y')} ➡ onwards"
```

Comment:
From the meetings yesterday it sounds like this will be simply to give them all an end date of `created_at + 1 year`. Wdyt?